package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "compendium-cdk",
+  "description": "Compendium for various stuff built on serverless stack",
+  "version": "0.0.2",
+  "module": "lib/index.js",
+  "main": "lib/index.js",
+  "author": {
+    "name": "Jakub Knejzlik",
+    "email": "jakub.knejzlik@gmail.com"
+  },
+  "dependencies": {
+    "@aws-cdk/aws-appsync-alpha": "^2.55.1-alpha.0",
+    "@aws-sdk/types": "^3.127.0",
+    "@tsconfig/node16": "^1.0.1",
```

Comment:
What version of node do we want to use? We are using node 16 ts config and node 18 types. Let's use same version for whole project

Reply:
node 18 is used for whole project